### ISSUE TITLE
con't find data/metaylo.data file

### ISSUE THREAD

**Issue by Jinzifeng** [bingykang/Fewshot_Detection#48]
I cound't find the metayolo.data file in data directory,but i found it in the cfg directory,should i copy this file from cfg to data?

**Comment by terasakisatoshi** [comment]
Yep I think we need fix `cfg/metayolo.data` accordingly not `data/metayolo.data`
